### ISSUE TITLE
What about a prebuilt binaries repo?

### ISSUE THREAD

**Issue by 3oheicrw** [brechtsanders/winlibs_tools#2]
Instead of requiring us to download, setup and build everything ourselves. What about you having a build server that offer prebuilt binaries? So it will be very similar to MSYS2's pacman. The facility needed for this should be already available on winlibs_tools isn't it?

**Comment by brechtsanders** [comment]
Yes, that is what https://github.com/brechtsanders/winlibs_tools/ and https://github.com/brechtsanders/winlibs_recipes are for. I'm still working on some issues to get this up to production quality, but the good news is I'm already able to build over 2500 software packages.
